test/services/TextBuilderSpec.scala

```diff
@@ -95,63 +121,63 @@ class TextBuilderSpec extends BaseSpec {
 
       val text = ""
 
-      TextBuilder.fragmentsToDisplay( text ) mustBe text
+      BulletPointBuilder.fragmentsToDisplay( text ) mustBe text
```

Comment:
The tests for BulletPointBuilder.fragmentsToDisplay should really be in BulletPointBuilderSpec.

Reply:
Yes agree, I'll move them and push to branch when done